src/plugins/ExportKeras/ExportKeras.js

```diff
@@ -0,0 +1,73 @@
+/*globals define*/
+/*eslint-env node*/
+define([
+    'plugin/PluginConfig',
+    'text!./metadata.json',
+    'SimpleNodes/Constants',
+    'plugin/GenerateKeras/GenerateKeras/GenerateKeras'
+], function (
+    PluginConfig,
+    pluginMetadata,
+    SimpleConstants,
+    PluginBase) {
+    'use strict';
+
+    pluginMetadata = JSON.parse(pluginMetadata);
+    const { exec } = require('child_process');
+    function ExportKeras() {
+        PluginBase.call(this);
+        this.pluginMetadata = pluginMetadata;
+        this.outputFileName = '';
+    }
+
+    ExportKeras.metadata = pluginMetadata;
+
+    // Prototypical inheritance from GenerateKeras.
+    ExportKeras.prototype = Object.create(PluginBase.prototype);
+    ExportKeras.prototype.constructor = ExportKeras;
+
+    ExportKeras.prototype.createOutputFiles = async function(activeNode){
+        const files = PluginBase.prototype.createOutputFiles.call(this, activeNode);
+        if(!this.outputFileName){
+            let fileName = this.getCurrentConfig().jsonFileName || 'exported_architecture.json';
+            if(!fileName.endsWith('.json')){
+                fileName = `${fileName}.json`;
+            }
+            this.outputFileName = fileName;
+        }
+
+        let outputFiles = {};
+        let codeString = files['output.py'];
+        const modelJSONName = this.generateVariableName('model_json');
+        let code = codeString.split('\n');
+
+        code.push('');
+        code.push(`${modelJSONName} = ${this.RESULT_VARIABLE_NAME}.to_json()`);
+        code.push(`print(${modelJSONName})`);
+        outputFiles[this.outputFileName] = await this.executePython(code.join('\n'));
+        this.logger.info(`Exported JSON Model filename is ${this.outputFileName}`);
+        return outputFiles;
+    };
+
+    ExportKeras.prototype.executePython = function (code, pythonVersion='3.5') {
```

Comment:
Should we set `pythonVersion` to `3` by default (rather than `3.5`)? I am not suggesting that we change the actual recommended version but thinking it might be nice to be consistent with the [environment test](https://github.com/deepforge-dev/deepforge-keras/blob/master/test/requirements/python-requirements.spec.js#L11). It also might be nice not to completely fail if the user isn't using the recommended version of python.

Reply:
Agreed. I have changed python version to 3. Throw an error if any other version than `python3` is used.